profile.html

```diff
@@ -0,0 +1,71 @@
+<!doctype html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7" lang=""> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8" lang=""> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9" lang=""> <![endif]-->
+<!--[if gt IE 8]><!--> <html class="no-js" lang=""> <!--<![endif]-->
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+  <title></title>
+  <meta name="description" content="">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="apple-touch-icon" href="apple-touch-icon.png">
+
+  <link rel="stylesheet" href="css/bootstrap.min.css">
+  <style>
+    body {
+      padding-top: 50px;
+      padding-bottom: 20px;
+    }
+  </style>
+  <link rel="stylesheet" href="css/bootstrap-theme.min.css">
+  <link rel="stylesheet" href="css/main.css">
+
+  <script src="js/vendor/modernizr-2.8.3-respond-1.4.2.min.js"></script>
+</head>
+
+<body>
+
+<!--TODO Update to use navbar from other branch-->
+<nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
+  <div class="container">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <a class="navbar-brand" href="#">Project name</a>
+    </div>
+    <div id="navbar" class="navbar-collapse collapse">
+      <form class="navbar-form navbar-right" role="form">
+        <button type="submit" class="btn btn-success">Sign Out</button>
+      </form>
+    </div><!--/.navbar-collapse -->
+  </div>
+</nav>
+
+<div>
+<h2 class="col-xs-1 col-xs-offset-5">Profile</h2>
+</div>
+<!--<div class="input-group col-sm-6 col-sm-offset-1">-->
+  <!--<input type="text" class="form-control" placeholder="What's your story?" aria-describedby="basic-addon2">-->
+  <!--<span class="input-group-addon" id="basic-addon2">@example.com</span>-->
+<!--</div>-->
+
+<h4 class="col-xs-8 col-xs-offset-2 col-md-6">About</h4>
+
+<form action="./poems.html" method="get" role="form" class="form-horizontal">
+  <div class="form-group">
+    <div class="col-xs-8 col-xs-offset-2 col-md-6">
+      <textarea class="form-control" rows="3" placeholder="What's your story? What poems do you like?" required></textarea>
+    </div>
+  </div>
+  <div class="save-btn col-xs-8 col-xs-offset-2 ">
+  <button type="button" class="btn btn-default">Save</button>
+  </div>
```

Comment:
We could also look to implement a "cancel" button next to the save button that would clear out the text the user entered into the form if they decide at the last minute not to add it to their bio. This would be a implementation for a future sprint though not something that needs to be added to this issue as this is just about adding a save button.